static/app/views/projectInstall/platform.tsx

```diff
@@ -217,6 +217,7 @@ export function ProjectInstallPlatform({
                 organization,
                 platform: platform.name ?? 'unknown',
                 project_id: project.id,
+                products,
```

Comment:
Do we know how amplitude works when you send it an array of strings? E.g is it smart enough to be able to answer how many take me to issues where clicked with replay selected as one of the products? Or do we need to explode it into multiple booleans?

Reply:
+ does different order in the array increase cardinality?

Reply:
👍